cluster-management/setup/mounting-storage/index.md

```diff
@@ -64,6 +64,42 @@ coreos:
 
 Notice that we're starting both units at the same time and using the power of systemd to work out the dependencies for us. In this case, `var-lib-docker.mount` requires `format-ephemeral.service`, ensuring that our storage will always be formatted before it is mounted. Docker will refuse to start otherwise.
 
+## Creating and Mounting a btrfs volume file
+
+Those interested in using btrfs on CoreOS can do so with very little overhead simply by creating two systemd units: one that creates and formats a btrfs volume file and another that mounts it.
```

Comment:
Maybe add a little background here since we used to be btrfs for the root. Something like:

```
CoreOS versions 6xx.0.0 and later use ext4 + overlayfs to provide a layered filesystem for the root partition. Prior to this, CoreOS 6xx.0.0 and earlier used btrfs for this functionality. If you'd like to continue using btrfs on newer CoreOS machines, you can do so with two systemd units: one that creates and formats a btrfs volume file and another that mounts it.
```